docs/_layouts/docs.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE>
+<html lang="en-us">
+{% include head.html %}
+<body class="docs">
+    {% include navigation-dropdown.html %}
+    <header class="header">
+        <div class="container">
+            <div class="row">
+              <div class="col-md-8">
+                <div class="title-container">
+                    <h1 class="display-2 text-white font-condensed mb-4">{{ page.title }}</h1>
+                    {% if page.subtitle %}
+                        <span class="subtitle text-white heading-underline" style="text-transform: none;">{{ page.subtitle }}</span>
+                    {% endif %}
+                </div>
+              </div>
+              <div class="col-md-4">
+                <div class="card github">
+                  <div class="card-block">
+                    <h2 class="card-title">Want to contribute?</h2>
+                    <p class="card-text">If you're interested in contributing the Athena Project, please feel free to visit our GitHub Page and send us
```

Comment:
Let's update "Athena Project" references to "Athena Framework" to help differentiate between the two.